skore/tests/unit/sklearn/test_estimator.py

```diff
@@ -925,3 +927,48 @@ def test_estimator_report_get_X_y_and_data_source_hash(data_source):
         assert X is X_test
         assert y is y_test
         assert data_source_hash == joblib.hash((X_test, y_test))
+
+
+def test_estimator_has_side_effects():
```

Comment:
```suggestion
@pytest.mark.parametrize("prefit_estimator", [True, False])
def test_estimator_has_side_effects(prefit_estimator):
```

Reply:
Now, it should always be true. Let's check both prefitted and not fitted.

Reply:
done